onnxruntime/python/tools/quantization/processing.py

```diff
@@ -0,0 +1,135 @@
+import os
+import sys
+import numpy as np
+import re
+from PIL import Image
+import cv2
+import pdb
+
+def yolov3_preprocess_func(images_folder, height, width, start_index=0, size_limit=0):
+    '''
+    Loads a batch of images and preprocess them
+    parameter images_folder: path to folder storing images
+    parameter height: image height in pixels
+    parameter width: image width in pixels
+    parameter size_limit: number of images to load. Default is 0 which means all images are picked.
+    return: list of matrices characterizing multiple images
+    '''
+    # this function is from yolo3.utils.letterbox_image
```

Comment:
from where? do we need to add any license header?

Reply:
Add the source. https://github.com/qqwweee/keras-yolo3/blob/master/yolo3/utils.py